legal-api/src/legal_api/reports/report.py

```diff
@@ -1086,12 +1086,14 @@ def _format_correction_data(self, filing):
             versioned_business = VersionedBusinessDetailsService.\
                 get_business_revision_obj(prev_completed_filing.transaction_id, self._business.id)
 
-            self._format_name_request_data(filing, versioned_business)
-            self._format_name_translations_data(filing, prev_completed_filing)
-            self._format_office_data(filing, prev_completed_filing)
-            self._format_party_data(filing, prev_completed_filing)
-            self._format_share_class_data(filing, prev_completed_filing)
-            self._format_resolution_data(filing)
+            is_comment_only_correction = filing.get('correction', {}).get('commentOnly', '')
+            if not is_comment_only_correction:
```

Comment:
move this to the beginning of this func. commentOnly is generic to all legal types.